extensions/2.0/Vendor/DRACO_animation_compression/README.md

```diff
@@ -0,0 +1,236 @@
+            "samplers" : [
+                {
+                    "input" : 0
+                    "interpolation" : "LINEAR",
+                    "output" : 3
+                },
+                {
+                    "input" : 4
+                    "interpolation" : "LINEAR",
+                    "output" : 5
+                }
+            ]
+        },
+    ],
+    "accessors" : [
+        // Accessors of input/output of animations that use the compression extension will omit "bufferView" property.
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "SCALAR"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC3"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC4"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC4"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "SCALAR"
+        },
+        {
+            "componentType" : 5123,
+            "count" : 100,
+            "type" : "VEC3"
+        },
+        ...
+    ],
+    "bufferViews" : [
+        // Contains data for the first compressed animation.
+        {
+            "buffer" : 10,
+            "byteOffset" : 0,
+            "byteLength" : 10000
+        },
+        // Contains data for the second compressed animation.
+        {
+            "buffer" : 10,
+            "byteOffset" : 10000,
+            "byteLength" : 6000
+        },
+        ...
+    ]
+}
+
+
+```
+
+### JSON Schema
+
+For full details on the `Draco_animation_compression` extension properties, see the schema:
+
+* [extension property (the entirety of this specification is normative and included in Scope)](schema/node.DRACO_animation_compression.schema.json)
+
+## Conformance
+
+Below is the recommended process when a loader encounters a glTF asset with the Draco animation extension set:
+
+* If `Draco_animation_compression` is in `extensionsRequired` and the loader does not support the Draco animation extension, then the loader must fail loading the asset.
+* If the loader does not support the Draco animation extension and `Draco_animation_compression` is not in `extensionsRequired`, then load the glTF asset ignoring `Draco_animation_compression` in `extensions`.
+* If the loader does support the Draco animation extension, but will not process `Draco_animation_compression`, then the loader must load the glTF asset ignoring `Draco_animation_compression` in `extensions`.
+* If the loader does support the Draco animation extension, and will process `Draco_animation_compression` then:
+    * The loader must process `Draco_animation_compression` first. The loader must get the data from `Draco_animation_compression`'s `bufferView` property and decompress the data using a Draco animation decoder to a Draco animation.
```

Comment:
> decompress the data using a Draco animation decoder to a Draco animation.

Is there a 1:1 relationship between a `Draco_animation_compression` bufferView and an animation? Or could two glTF animations share a single bufferView but reference a different subset of available attributeIds?

Reply:
The later (for "outputs"). Actually what we are currently doing is we parse the source gltf file and we gather all outputs that correspond to the same input. Then we compress the the one input with all of it's output in one animation compression. That is why we have an array of "attributeId" and an array of "output".

For each unique input we create one animation compression.